src/content/concepts/configuration.md

```diff
@@ -30,7 +30,7 @@ T> The most important part to take away from this document is that there are man
 
 The examples below describe how webpack's configuration can be both expressive and configurable because _it is code_:
 
-## Simple Configuration
+## Introductory Configuration
```

Comment:
I think we need to update the redirects list here https://github.com/webpack/webpack.js.org/blob/master/webpack.ssg.js#L57, otherwise links to this heading would be break.

Reply:
@chenxsan I looked at this list of redirects and couldn't find anything related to these headings. The rest of the redirects listed there still appear to be in place. 

Happy to address it if you're able to add more detail. Thanks!

Reply:
Here's the link for current `Simple Configuration` heading `/concepts/configuration/#simple-configuration`, and the updated one would be `/concepts/configuration/#introductory-configuration`, hence we need to append an redirect item like:

```json
'/concepts/configuration/#simple-configuration': '/concepts/configuration/#introductory-configuration',
```

Reply:
@chenxsan ahh, I see. I was misunderstanding. Thank you for the clarification! I'll do this.